### DISCUSSION TITLE
请求添加 Minetest

### DISCUSSION THREAD

**Discussion by Pevernow** [Gamuxorg/bbs#263] [投递和请求游戏]
开源3D沙盒游戏，有完整的插件系统，更新非常积极
目前最新版本5.3.0
主页: https://minetest.net/
Github存储库：https://github.com/minetest/minetest

**Comment by shouhuanxiaoji** [comment]
好的，尽快收录

**Comment by shouhuanxiaoji** [comment]
正在编写编译脚本，以后会定期更新版本的

**Comment by shouhuanxiaoji** [comment]
完成
